src/clj/rems/application/master_workflow.clj

```diff
@@ -0,0 +1,189 @@
+(ns rems.application.master-workflow
+  "The master workflow is a superset of all possible commands and who is
+  allowed to execute them. Workflows for production use can be derived from
+  the master workflow by restricting the possible commands (permissions)."
+  (:require [rems.application.commands :as commands]
+            [rems.permissions :as permissions]))
+
+(def whitelist
+  (permissions/compile-rules
+   (into [{:permission :see-everything}]
+         (map (fn [command]
+                {:permission command})
+              (sort (keys commands/command-schemas))))))
+
+(defmulti calculate-permissions
```

Comment:
is `calculate-permissions` just moved from model.clj or are there changes?

Reply:
These are the only differences:

<img width="1489" alt="diff" src="https://user-images.githubusercontent.com/42678/70704568-e80f8180-1cda-11ea-956f-25acf853c22f.png">